lib/plugsnag/basic_error_report_builder.ex

```diff
@@ -55,6 +55,12 @@ defmodule Plugsnag.BasicErrorReportBuilder do
     |> Keyword.get(field, [])
   end
 
+  defp filter(:query_string, data) when is_binary(data) do
+    data
+    |> URI.decode_query
+    |> do_filter(filters_for(:params))
+    |> URI.encode_query
```

Comment:
Style: [use parentheses for one-arity functions when using the pipe operator](https://github.com/christopheradams/elixir_style_guide#parentheses-pipe-operator).

Reply:
Done.